cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -86,10 +91,8 @@ updated. Existing keys not in the properties map will not be deleted.
      - Description
    * - ``namespace-id``
      - Namespace ID
-   * - ``entity-type``
-     - One of ``apps``, ``datasets``, or ``streams``
-   * - ``entity-id``
-     - Name of the entity
+   * - ``entity-details``
+     - Hierarchical key-value representation of the entity
```

Comment:
Can you add an example? I don't know what this means by just reading this doc. Same for other places entity-details is used.

Reply:
Done. Resolved by linking the section down.